src/fides/api/ops/schemas/privacy_request.py

```diff
@@ -63,6 +63,14 @@ class Consent(BaseSchema):
     conflicts_with_gpc: bool = False
 
 
+class ConsentReport(Consent):
+    """Schema for reporting Consent requests."""
+
+    identity: Identity
```

Comment:
> There will not be any browser identities to show here, so I don't think it makes sense to include these keys:

I wanted to leave it extensible for us in the future, since I know we're planning on using these for some future onboards. It'd be nice to be able to add those and not to have to worry about updating them here.

Reply:
OK is my point clear though that this information is _not_ saved with respect to the Consent/ConsentRequest records though?  It's saved on the PrivacyRequest.

Reply:
Yes that's clear. We've talked about storing it on the `ProvidedIdentity` table too, which is what I'd like to leave the options open for.

Reply:
OK I am confused then because as far as I know the plan is still to phase out this workflow?  New work will not be writing to the Consent table at all.

Reply:
Thanks for the chat offline — I've removed these those two fields :)

Reply:
thank you @seanpreston !